.github/workflows/publish-website.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       - name: Publish ${{ github.ref }}
         run: sbt docs/docusaurusPublishGhpages
         env:
-          GITHUB_DEPLOY_KEY: ${{ secrets.GITHUB_DEPLOY_KEY }}
+          GIT_DEPLOY_KEY: ${{ secrets.GIT_DEPLOY_KEY }}
```

Comment:
I think it's no matter how we define the env var in secrets, but I thought it should be `GITHUB_DEPLOY_KEY` in CI as done in Scalafix, for example https://github.com/scalacenter/scalafix/blob/main/.github/workflows/deploy-website.yml#L17

Reply:
Yeah, I'm pretty confident it should be `GITHUB_DEPLOY_KEY` in CI https://github.com/search?q=org%3Ascalacenter%20GITHUB_DEPLOY_KEY&type=code

Reply:
Perhaps the mdoc website is a little bit outdated 🤷🏻

Reply:
can't think why it would matter what it's called as long as we're consistent 🤷